nft/righter.go

```diff
@@ -0,0 +1,75 @@
+package nft
+
+import (
+	"github.com/spikeekips/mitum/base"
+	"github.com/spikeekips/mitum/util"
+	"github.com/spikeekips/mitum/util/hint"
+	"github.com/spikeekips/mitum/util/isvalid"
+)
+
+var (
+	RighterType   = hint.Type("mitum-nft-righter")
+	RighterHint   = hint.NewHint(RighterType, "v0.0.1")
+	RighterHinter = Righter{BaseHinter: hint.NewBaseHinter(RighterHint)}
+)
+
+type Righter struct {
```

Comment:
@wyuinche What does the name `Righter` mean?

Reply:
@wyuinche There are cases where er is added when naming, but as far as I know, it is a case of creating an interface name by attaching it to the name of a method such as Print(), Printer Read(), Reader, etc. Of course, in the current mitum model, there are cases where a type is named by appending er even if it is not an interface, but the name Righter has an inaccurate meaning as well as does not seem to follow the convention.
If you can't think of a good name right now, set it to RightHolder first and change it later.

Reply:
@soonkuk `Righter` has been renamed `RightHolder`.